packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -1355,6 +1355,61 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
                     hc: '#707070'
                 },
                 description: 'ANSI white background color'
+            },
+
+            // Theia defaults
+            // Buttons
+            {
+                id: 'button.secondary.foreground',
```

Comment:
`button.secondaryForeground` or `secondaryButton.foregrond`

We will define naming later to color variables, a pattern should be `object`.`property`.